terminus-ui/csv-entry/src/csv-entry.component.html

```diff
@@ -94,7 +94,7 @@
       <!-- Column for delete buttons -->
       <div fxLayout="column">
         <!-- Fake row for header -->
-        <div style="height: 47px" fxFlex="noshrink"></div>
+        <div [style.height]="fakeRowHeight"  fxFlex="0 0 auto"></div>
```

Comment:
cool... didn't know you could do this....

Reply:
Yeah, it somehow felt a _little_ less dirty than inlining the style directly..